missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/GameListener.java

```diff
@@ -212,37 +183,48 @@ public void onJoin(PlayerArenaJoinEvent event) {
     public void onThrow(ProjectileLaunchEvent event) {
         if (!isInGameWorld(event.getEntity().getLocation())) return;
 
-        Game game = getGame();
         if (event.getEntity() instanceof Snowball) {
-            Snowball ball = (Snowball) event.getEntity();
-            if (ball.getShooter() instanceof Player) {
-                Shield shield = new Shield((Player) ball.getShooter(), game.getArena().getShieldConfiguration());
-                shield.onThrow(event);
-            }
+            Snowball snowball = (Snowball) event.getEntity();
+
+            if (!(snowball.getShooter() instanceof Player)) return;
+
+            Player shooter = (Player) snowball.getShooter();
+            Shield shield = new Shield(shooter, getGame().getArena().getShieldConfiguration());
+            shield.onThrow(event);
         }
     }
 
     @EventHandler
     public void onDamage(EntityDamageByEntityEvent event) {
-        if (!isInGameWorld(event.getEntity().getLocation())) return;
         if (!(event.getEntity() instanceof Player)) return;
-      
-        Player p = (Player) event.getEntity();
+
+        Player player = (Player) event.getEntity();
+        if (!isInGameWorld(player.getLocation())) return;
+
+        Player shooter;
         if (event.getDamager() instanceof Projectile) {
-            Projectile pj = (Projectile) event.getDamager();
-            Player shooter = (Player) pj.getShooter();
-            if (Objects.requireNonNull(getGame().getPlayer(shooter)).getTeam() == Objects.requireNonNull(getGame().getPlayer(p)).getTeam()) {
-                shooter.sendMessage(MessageConfig.getMessage("hurt_teammates"));
-                event.setCancelled(true);
-            }
-            return;
-        }
-        if (event.getDamager() instanceof Player) {
-            Player d = (Player) event.getDamager();
-            if (Objects.requireNonNull(getGame().getPlayer(d)).getTeam() == Objects.requireNonNull(getGame().getPlayer(p)).getTeam()) {
-                d.sendMessage(MessageConfig.getMessage("hurt_teammates"));
-                event.setCancelled(true);
-            }
+            Projectile projectile = (Projectile) event.getDamager();
+
+            if (!(projectile.getShooter() instanceof Player)) return;
+
+            shooter = (Player) projectile.getShooter();
+
+        } else if (event.getDamager() instanceof Player) {
+
+            shooter = (Player) event.getDamager();
+
+        } else return;
```

Comment:
Please use here also brackets. If one part of the if clause uses brackets, then the other parts should too.
```suggestion
        } else {
          return;
        }
```